From cc8618aa381c26c3aaeb40ac9617adeaac14dbfc Mon Sep 17 00:00:00 2001
From: Magnus Damm <damm@opensource.se>
Date: Mon, 14 May 2012 19:54:51 +0900
Subject: ARM: Undelete KZM9D mach-type

Undelete the KZM9D mach-type to allow build of board
for EMEV2 SoC support.

Signed-off-by: Magnus Damm <damm@opensource.se>

N.B: Not present upstream

Signed-off-by: Simon Horman <horms@verge.net.au>
---
 arch/arm/tools/mach-types |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/tools/mach-types b/arch/arm/tools/mach-types
index 2997e56..e2e0d3f 100644
--- a/arch/arm/tools/mach-types
+++ b/arch/arm/tools/mach-types
@@ -462,6 +462,7 @@ msm7x27_surf		MACH_MSM7X27_SURF	MSM7X27_SURF		2705
 msm7x27_ffa		MACH_MSM7X27_FFA	MSM7X27_FFA		2706
 msm7x30_ffa		MACH_MSM7X30_FFA	MSM7X30_FFA		2707
 qsd8x50_surf		MACH_QSD8X50_SURF	QSD8X50_SURF		2708
+kzm9d			MACH_KZM9D		KZM9D			3246
 mx53_evk		MACH_MX53_EVK		MX53_EVK		2716
 igep0030		MACH_IGEP0030		IGEP0030		2717
 sbc3530			MACH_SBC3530		SBC3530			2722
-- 
1.7.10.1.362.g242cab3

